recipe/build.py

```diff
@@ -352,12 +352,13 @@ def extract(self):
                 # "--nox11" runfile command prevents desktop GUI on local install
                 cmd = [
                     os.path.join(self.src_dir, self.runfile),
-                    "--installpath=%s" % (tmpd),
+                    f"--installpath={tmpd}",
                     "--toolkit",
                     "--silent",
                     "--override",
                     "--nox11",
                 ]
+                print(f"Extract command: {' '.join(cmd)}")
```

Comment:
Do we want to leave this print in here?

Reply:
It doesn't hurt. It helps with debugging

Reply:
Yeah I guess we don't see what command was called if we remove this right? Seems worth keeping in that case.